clib/table.ml

```diff
@@ -0,0 +1,138 @@
+(************************************************************************)
```

Comment:
Umm, maybe this should go into a private library for bench use only?

Reply:
I tried this but failed to get a shared library. Also the current ocaml scripts work out-of-the-box, I have no idea how to make shared private libraries do this.

Reply:
just turn the script into an executable, and use `dune exec -- `  as we do for configure. This way we get better editor support, etc....